src/UI/Components/Diagram/actions.ts

```diff
@@ -177,10 +189,13 @@ export function appendEmbeddedEntity(
     return createdInstances;
   } else {
     const instanceAsTable = new ServiceEntityBlock()
-      .setTabColor("#0066CC")
+      .setTabColor("#009596")
```

Comment:
it would be a nice idea to extract these into variables, also easier to read what color it is then :) You can make an object named `Colors` maybe?

Reply:
Good point, Done :)